packages/gestalt/src/RadioGroupButton.tsx

```diff
@@ -130,27 +140,78 @@ const RadioGroupButtonWithForwardRef = forwardRef<HTMLInputElement, Props>(funct
     borderWidth = styles.BorderCheckedMd;
   }
 
+  const uncheckedBorderWidth = disabled ? styles.BorderDisabled : styles.BorderUnchecked;
+
   const styleSize = size === 'sm' ? controlStyles.sizeSm : controlStyles.sizeMd;
 
   const bgStyle = disabled && !checked ? styles.BgDisabled : styles.BgEnabled;
 
   const { isFocusVisible } = useFocusVisible();
 
   const { parentName } = useRadioGroupContext();
+
+  const tapScaleAnimation = useTapScaleAnimation();
+
   if (parentName !== 'RadioGroup') {
     throw new Error(
       `RadioGroup.RadioButton must be used within a [RadioGroup](https://gestalt.pinterest.systems/web/radiogroup).`,
     );
   }
 
+  const radioButtonStyles = isInVRExperiment
+    ? classnames(styleSize, layoutStyles.relative, borderStyles.circle, tapScaleAnimation.classes, {
+        [focusStyles.accessibilityOutlineFocus]: focused && isFocusVisible,
+      })
+    : classnames(bgStyle, borderColor, borderWidth, styleSize, styles.RadioButton, {
+        [focusStyles.accessibilityOutlineFocus]: focused && isFocusVisible,
+      });
+
+  const checkedStyles = classnames(
+    styles.VRRadioButton,
```

Comment:
can we dry this so we know which props are reused in unchecked and checked and which ones are for one or the other.

   p.e. all this are shared:
styles.VRRadioButton,
borderColor,
 bgStyle,
    styleSize,